cmd/cluster/openstack/create.go

```diff
@@ -0,0 +1,189 @@
+package openstack
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/openshift/hypershift/cmd/util"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	hyperv1 "github.com/openshift/hypershift/api/hypershift/v1beta1"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func DefaultOptions() *CreateOptions {
+	return &CreateOptions{}
+}
+
+func BindOptions(opts *CreateOptions, flags *pflag.FlagSet) {
+	flags.StringVar(&opts.OpenStackCredentialsFile, "openstack-credentials-file", opts.OpenStackCACertFile, "Path to the OpenStack credentials file (required)")
+	flags.StringVar(&opts.OpenStackCACertFile, "openstack-ca-cert-file", opts.OpenStackCACertFile, "Path to the OpenStack CA certificate file (optional)")
+	flags.StringVar(&opts.OpenStackExternalNetworkName, "openstack-external-network-name", opts.OpenStackExternalNetworkName, "Name of the OpenStack external network (required)")
```

Comment:
this should be not required right? we can even disable the external network later on.

Reply:
indeed